packages/cli/src/commands/account/new.ts

```diff
@@ -38,13 +39,23 @@ export default class NewAccount extends LocalCommand {
       description:
         "Language for the mnemonic words. **WARNING**, some hardware wallets don't support other languages",
     }),
+    mnemonic: flags.string({
+      description:
+        'Instead of generating a new mnemonic (seed phrases) the user can set the mnemonic to be used. It is required to set it as a string with all the words in order, separated by spaces (example: "word1 word2 word3 ... word24"). If the words are in other language than enaglish, the --language flag must be used. Should be a bip39 mnemonic',
+    }),
+    derivationPath: flags.string({
+      hidden: true,
```

Comment:
I think it may as well be shown - what’s the value in hiding it from a user ? Also I think your warning is a little scary. Maybe “Recreating the same account requires knowledge of both the mnemonic and derivation path”

Reply:
Haha, yes, my idea was to be scary to avoid non technical users, but I can change it